src/main/java/somethingrandom/dataaccess/google/tasks/GoogleTasksDataAccessObject.java

```diff
@@ -19,6 +25,36 @@ public GoogleTasksDataAccessObject(TaskList list) {
 
     @Override
     public void save(Item item) throws DataAccessException {
-        throw new RuntimeException("not implemented");
+        String id = taskList.getIdentifier();
+        try {
+            JSONObject request = new JSONObject();
+            request.put("kind", "tasks#task");
+            request.put("id", item.getID());
+            request.put("title" , item.getName());
+            request.put("status" , "needsAction");
+
+            //To keep track of creation date because Google tasks only tracks the last updated time
+            //Probably going to change this later, feel free to change or suggest a better way to do this
+            String notes = "Created on " + item.getCreationDate();
+            if (item.getItemKind() == ItemKind.ACTIONABLE){
+                if (((ActionableItem) item).getNeededTime() != null) {
+                    request.put("status" , "needsAction");
+                    notes = notes + "\n" + "Needs " + ((ActionableItem) item).getNeededTime().toHours() + " hours to complete";
+                } else {
+                    notes = notes + "\n" + "Needs to be completed";
+                }
+            } else if (item.getItemKind() == ItemKind.DELAYED){
+                request.put("status" , "needsAction");
+                notes = notes + "\n" + "Reminder set on " + ((DelayedItem) item).getReminderDate();
+            } else {
+                notes = notes + "\n" + ((ReferenceItem) item).getDescription();
+            }
+            request.put("notes" , notes);
+
+            this.taskList.getProvider().request(new APIRequestBody.JSONBody("POST",request), "https://tasks.googleapis.com/tasks/v1/lists/" + id + "/tasks");
+
+        } catch (IOException e) {
+            throw new DataAccessException(e);
+        }
```

Comment:
Ideally, the try/catch would only surround the `.request` call, but it's not too important.